### ISSUE TITLE
wrong Oracle outer join conversion

### ISSUE THREAD

**Issue by glr72** [jOOQ/jOOQ#12591]
### Expected behavior

SELECT   *
       FROM   
              t1 SLC,
              t2 CHG,
              t3 CGS,
              t4 GRS,
              t5 CHL,
              t6 LIN,
              t7 SLA,
              t8 SLL
      WHERE       SLC.c1 = CHG.c2(+)
              AND CHG.c3 = CGS.c4(+)
              AND CGS.c5 = GRS.c6(+)
              AND CGS.c7 = CHL.c8(+)
              AND SLC.c9 = SLA.c10(+)
              AND SLA.c11 = SLL.c12(+)
              AND SLL.c13 = LIN.c14(+)
              AND CHL.c15 = LIN.c16

in ANSI rewrite, call tables must be unique

### Actual behavior
Table SLL is defined twice.

SELECT *
FROM T1 SLC
  LEFT OUTER JOIN T2 CHG
    ON SLC.C1 = CHG.C2
  LEFT OUTER JOIN T3 CGS
    ON CHG.C3 = CGS.C4
  LEFT OUTER JOIN T4 GRS
    ON CGS.C5 = GRS.C6
  LEFT OUTER JOIN T7 SLA
    ON SLC.C9 = SLA.C10
  LEFT OUTER JOIN T8 SLL.  <--- first def
    ON SLA.C11 = SLL.C12
  LEFT OUTER JOIN (
    T5 CHL
      JOIN (
        T6 LIN
          RIGHT OUTER JOIN T8 SLL <--- second def
            ON SLL.C13 = LIN.C14
      )
        ON CHL.C15 = LIN.C16
  )
    ON CGS.C7 = CHL.C8
### Versions
https://www.jooq.org/translate/ (Oracle to Postgres)

**Comment by lukaseder** [comment]
Thanks a lot for your report. I can reproduce that. Will investigate right away what went wrong here.

**Comment by lukaseder** [comment]
This is a more minimal example with:

- Aliasing removed
- Unnecessary indirections on the left join path removed

```sql
SELECT *
FROM
  t1,
  t5,
  t6,
  t8
WHERE t1.c1 = t5.c8(+)
AND t1.c9 = t8.c12(+)
AND t8.c13 = t6.c14(+)
AND t5.c15 = t6.c16
```

It produces:

```sql
select *
from T1
  left outer join T8
    on T1.C9 = T8.C12
  left outer join (
    T5
      join (
        T6
          right outer join T8
            on T8.C13 = T6.C14
      )
        on T5.C15 = T6.C16
  )
    on T1.C1 = T5.C8
```

While the result is clearly wrong, I'm a bit confused about this predicate here:

```sql
AND t5.c15 = t6.c16
```

It effectively turns the entire join graph into a set of inner joins, if I'm not mistaken? Can you explain the intent of this query?

**Comment by glr72** [comment]
I'm not sure about the predicate

AND t5.c15 = t6.c16

I simplified a query provided by some test cases, I didn't ask for the business reason

**Comment by lukaseder** [comment]
Yeah, irrespective of this bug here, I think that looks like a bug in the query...

**Comment by glr72** [comment]
I see, I think we have bugs on both sides.
